xml_inscriptions/metadata_only/IL.Chi.UC.SMA.G.1978.18.xml

```diff
@@ -84,7 +84,7 @@
 
                         <origin>
                             <date evidence="unspecified" notBefore="0099" notAfter="0200">Second century CE</date>
-                            <placeName ref="xx">Detailed place name</placeName>
+                            <placeName ref="unknown"/>
```

Comment:
You still need to have something within the `<placeName>` element, otherwise there will be nothing for the system to display.